waypoints_eepose:
  home:
  - 0.198
  - 0.0
  - 0.19305
  - 0.0
  - 0.0
  - 0.0
  - 1.0
  sleep:
  - 0.04448547222590873
  - -3.829587735405626e-06
  - 0.10736444468414988
  - 8.97229236117845e-06
  - 0.20844889369325448
  - -4.209761053873285e-05
  - 0.978033259590566
  pre_place:
  - -0.09088042843036702
  - -0.16216198557434486
  - 0.12516752520381097
  - 0.5764809158781462
  - 0.33776253496713005
  - -0.6419611958940832
  - 0.3761276998136541
  place:
  - -0.09016691669094565
  - -0.1744412172141128
  - 0.0751172864633296
  - 0.6095152650251681
  - 0.3710721689942466
  - -0.5983974041884702
  - 0.3643036285294245
  home_candle:
  - -0.008951734549797643
  - -0.14214433355166278
  - 0.20124171262677099
  - 0.3529755620872543
  - 0.3314457035360419
  - -0.637835545165635
  - 0.59893055983139
  pre_grab:
  - -0.024613075752831226
  - -0.1854887692097508
  - 0.11166840728742633
  - 0.5190741148383752
  - 0.45474641887131534
  - -0.5443639476386931
  - 0.47690213916451424
  grab:
  - -0.02837570835257389
  - -0.19129335067015885
  - 0.08310884573956791
  - 0.5391107801708654
  - 0.4650401083719336
  - -0.5317187745551666
  - 0.45866372114495035
  makis:
  - 0.10569573640531754
  - -0.0008106920769395333
  - 0.14622047975194652
  - 0.001831082837346828
  - 0.47746980837315545
  - -0.003369558045302898
  - 0.8786397870039373
  sakis:
  - 0.1405000826444656
  - -0.0015087290701853962
  - 0.0799012692075107
  - 0.003626843862441484
  - 0.6755176417315052
  - -0.0039586827436424755
  - 0.7373242777394748

